facebook/OpenBIC
# common: ipmi: Add oem command to access BMC via ipmb

Summary:
- Add oem command NetFn:0x38 Cmd:0x6A to access BMC ipmb.
  - Request:
    - Byte1~3 IANA
    - Byte4 BMC's netfn
    - Byte5 BMC's cmd
    - Byte6~ BMC's data
  - Response:
    - Byte1 BIC's completion code
    - Byte2~4 IANA
    - Byte5 BMC's completion code
    - Byte6~ BMC's response data
  
Test Plan:
- Build Code: PASS
- Check command in BMC/BIC console: PASS

Log:
- BMC console:
  ```
  Check bmc ip, should get 10.10.12.145
  root@bmc-oob:~# bic-util mb 0xe0 0x6a 0x15 0xa0 0x00 0x0c 0x02 0x00 0x03
  15 A0 00 00 11 0A 0A 0C 91
  ```
- BIC console:
  ```
  Check bmc ip, should get 10.10.12.145
  uart:~$ platform ipmi raw 0x38 0x6a 0x15 0xa0 0x00 0x0c 0x02 0x00 0x03
  BIC self command netfn:0x38 cmd:0x6a response with cc 0x0
  00000000: 15 a0 00 00 11 0a 0a 0c  91
  ```
